### PR TITLE
Add and fix some documentations and a minor optimization

### PR DESCRIPTION
**Pull request by lovesh**
Documentation updates
- Code comment describing what some of the symbols from the paper are renamed to
- Fix name and comment on a test

Minor optimization reducing the input size to an MSM

Signed-off-by: lovesh <lovesh.bond@gmail.com>

### CODE REVIEW

**Review by Pratyush — APPROVED** [review]
Thanks for the PR!

**Review by ValarDragon — APPROVED** [review]
Nice catch!

**Comment by Pratyush** [comment]
Could you add a CHANGELOG entry, and then we can merge this!

**Comment by lovesh** [comment]
@Pratyush Done.

File: src/generator.rs [hunk 1]
```diff
@@ -218,12 +227,7 @@ where
 
     // Compute the L-query
     let l_time = start_timer!(|| "Calculate L");
-    let l_query = FixedBase::msm::<E::G1>(
-        scalar_bits,
-        g1_window,
-        &g1_table,
-        &l[num_instance_variables..],
-    );
+    let l_query = FixedBaseMSM::multi_scalar_mul::<E::G1>(scalar_bits, g1_window, &g1_table, &l);
```
Comment (Pratyush) [inline src/generator.rs:230]:
```suggestion
    let l_query = FixedBase::multi_scalar_mul::<E::G1>(scalar_bits, g1_window, &g1_table, &l);
```

**Pull request merged**